server/py/framework/db/sqldb/helpers.py

```diff
@@ -67,11 +67,11 @@ def update_labels(obj, labels: dict):
     old = {label.name: label for label in obj.labels}
     obj.labels.clear()
     for name, value in labels.items():
+        error_msg = f" of `{name}` label is too long. Maximum allowed length is {max_str_length} characters."
+        if len(str(name)) > max_str_length:
+            raise mlrun.errors.MLRunInvalidArgumentError("Name" + error_msg)
         if len(str(value)) > max_str_length:
-            raise mlrun.errors.MLRunInvalidArgumentError(
-                f"Value of `{name}` label is too long. "
-                f"Maximum allowed length is {max_str_length} characters."
-            )
+            raise mlrun.errors.MLRunInvalidArgumentError("Value" + error_msg)
```

Comment:
While this works, I'd use another helper function like this:
```python
def validate_label_length(label_type: str, label_name: str, label_value: str, max_length: int):
    """Validates the length of a label name or value and raises an error if it exceeds max_length."""
    if len(label_value) > max_length:
        raise mlrun.errors.MLRunInvalidArgumentError(
            f"{label_type} of `{label_name}` label is too long. Maximum allowed length is {max_length} characters."
        )
```
Then call twice:
```python
    validate_label_length("Name", name, name, max_str_length)
    validate_label_length("Value", name, value, max_str_length)
```